calculator/src/main/java/edu/nextstep/camp/calculator/CalculatorViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package edu.nextstep.camp.calculator
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import edu.nextstep.camp.calculator.domain.Calculator
+import edu.nextstep.camp.calculator.domain.Expression
+import edu.nextstep.camp.calculator.domain.Operator
+
+class CalculatorViewModel : ViewModel() {
+    private val calculator = Calculator()
+    private var expression = Expression.EMPTY
+
+    private val _text = MutableLiveData("")
+    val text: LiveData<String> get() = _text
```

Comment:
```suggestion
    val text: LiveData<String> = _text
```
get()을 사용하지 않고 바로 할당해도 무방합니다.